CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   during sign-in.
 - Added ip access control based on the `allow_access_from` field of `Account`
   during sign-in.
+- Added `AgentManager::update_config` method to notify agents to update their
+  configurations.
```

Comment:
I don't think simply using the term "notify" would suffice because we not only notify agents but also send the new configuration.

Reply:
I was wrong. The manager notifies the agents, and they fetch the configuration. Sorry for the disturbance.